accelerator/config.c

```diff
@@ -119,6 +142,96 @@ status_t ta_config_default_init(ta_config_t* const info, iota_config_t* const ic
   return ret;
 }
 
+status_t ta_config_file_init(ta_core_t* const conf, int argc, char** argv) {
+  int key = 0;
+  status_t ret = SC_OK;
+  struct option* long_options = cli_build_options();
+  yaml_parser_t parser;
+  yaml_token_t token;
+  FILE* file = NULL;
+  char* arg = NULL;
+  int state = 0;
+
+  // Initialize default configuration file path with '\0'
+  memset(conf->conf_path, '\0', sizeof(conf->conf_path));
```

Comment:
I think just initialization `conf->conf_path[0]` into `'\0'` is OK for initializing a string